### PR TITLE
Replace travis with github actions

### PR DESCRIPTION
**Pull request by luca-rath**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | https://github.com/sulu/sulu/issues/5693
| License | MIT
| Documentation PR | sulu/sulu-docs#prnum

#### What's in this PR?

Replace travis with github actions

### CODE REVIEW

**Comment by luca-rath** [comment]
See CI results here https://github.com/luca-rath/sulu-minimal/pull/1

File: README.md [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 [![GitHub license](https://img.shields.io/github/license/sulu/sulu-minimal.svg)](https://github.com/sulu/sulu-minimal/blob/1.6/LICENSE)
 [![GitHub tag (latest SemVer)](https://img.shields.io/github/tag/sulu/sulu-minimal.svg)](https://github.com/sulu/sulu-minimal/releases)
-[![GitHub tag (latest by date)](https://img.shields.io/github/tag-date/sulu/sulu-minimal.svg)](https://github.com/sulu/sulu-minimal/releases)
-[![Travis](https://travis-ci.org/sulu/sulu-minimal.png?branch=1.6)](https://travis-ci.org/sulu/sulu-minimal)
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/sulu/sulu-minimal/Test%20Application/1.6?label=test-workflow)](https://github.com/sulu/sulu-minimal/actions)
```
Comment (niklasnatter) [inline README.md]:
```suggestion
[![Test workflow status](https://img.shields.io/github/workflow/status/sulu/sulu-minimal/Test%20Application/1.6?label=test-workflow)](https://github.com/sulu/sulu-minimal/actions)
```

File: .github/workflows/test-application.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+            - name: Create database
+              run: |
+                  bin/adminconsole doctrine:database:create
+                  bin/adminconsole doctrine:database:create --env test
+
+            - name: Build sulu
+              run: bin/adminconsole sulu:build dev --no-interaction
+
+            - name: Lint
```
Comment (niklasnatter) [inline .github/workflows/test-application.yaml]:
```suggestion
            - name: Lint code
```

File: .github/workflows/test-application.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+              run: |
+                  bin/adminconsole lint:twig app/Resources/views
+                  bin/adminconsole lint:yaml app/config
+
+            - name: Run tests
```
Comment (niklasnatter) [inline .github/workflows/test-application.yaml]:
```suggestion
            - name: Execute test cases
```

**Review by niklasnatter — APPROVED** [review]

**Pull request merged**